### ISSUE TITLE
definitions.charge.properties.source has an odd type

### ISSUE THREAD

**Issue by dwwoelfel** [stripe/openapi#20]
The [charge.source](https://github.com/stripe/openapi/blob/master/openapi/spec2.json#L2103) property is {"$ref": "#/definitions/account_debit_account"}, should that be some other type, e.g. `card` or a oneOf type?

**Comment by brandur-stripe** [comment]
Hey @dwwoelfel, it should, but OpenAPI 2.0 doesn't support `oneOf` and its like in a few places so the spec we produce for that version has a number of inaccuracies.

This has been addressed for OpenAPI 3.0 which has broader support for things. Can you see if you could use that instead? Here's `charge.source` for example:

https://github.com/stripe/openapi/blob/master/openapi/spec3.yaml#L1483-L1491

**Comment by pakrym-stripe** [comment]
`source` is now defined as:

```
   anyOf:
          - "$ref": "#/components/schemas/payment_source"
```
